cmd/olm/main.go

```diff
@@ -106,6 +123,27 @@ func main() {
 	http.Handle("/metrics", prometheus.Handler())
 	go http.ListenAndServe(":8080", nil)
 
-	_, done := operator.Run(stopCh)
+	ready, done := operator.Run(stopCh)
+	<-ready
```

Comment:
Is it necessary to set the status to failed if ready doesn't happen in a certain amount of time?

Reply:
I think that CVO will essentially do that for you (if ClusterStatus isn't ready after the global timeout, install failed)